addons/mail/static/src/core/common/composer.js

```diff
@@ -134,10 +134,15 @@ export class Composer extends Component {
         });
         useExternalListener(window, "beforeunload", this.saveContent.bind(this));
         if (this.props.dropzoneRef) {
-            useCustomDropzone(this.props.dropzoneRef, MailAttachmentDropzone, {
-                extraClass: "o-mail-Composer-dropzone",
-                onDrop: this.onDropFile,
-            }, () => this.allowUpload);
+            useCustomDropzone(
```

Comment:
Useless diff or is there any change ?

Reply:
These LOCs were not linted properly, this is automatic diff when the linter is correctly set up.

Reply:
That makes the bugfix hard to read, as it mixes both cosmetic and real changes.

Reply:
That's the config of the JS linter: save on file applies the linter. If everyone have them properly set this is never a problem.

Reply:
Of *your* linter. Auto changing files has never been an official R&D guideline :) . Official R&D guideline is: minimal diff in stable when possible.